velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,49 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp =
+      [&](const std::string& functionName,
+          int64_t expectedTimestamp,
+          const std::string& expectedTimeZone) {
+        auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+        auto callExpr = std::make_shared<const core::CallTypedExpr>(
+            TIMESTAMP_WITH_TIME_ZONE(),
+            std::vector<core::TypedExprPtr>{},
+            functionName);
+
+        auto actual = evaluate(callExpr, emptyRowVector);
```

Comment:
Did you consider inspecting actual variable directly ? It should be of TimestampWithTimezone type and should have milliSeconds_ and timezoneId_ fields. Might be simpler to just compare those instead of constructing a vector here. Did you have any particular reason to construct this vector ?

Reply:
The main reason is that I saw other tests with output type TimestampWithTimezone in the DateTimeFunctionsTest.cpp are all comparing the vectors. So I follow the format. Also, the existing function there makes it easier to call the expression.